rmw_connext_dynamic_cpp/src/macros.hpp

```diff
@@ -51,11 +51,11 @@
         "type support handle implementation '%s' (%p) does not match valid type supports " \
         "('%s' (%p), '%s' (%p))", \
         TYPE_SUPPORTS->typesupport_identifier, \
-        TYPE_SUPPORTS->typesupport_identifier, \
-        rosidl_typesupport_introspection_cpp::typesupport_identifier, \
+        reinterpret_cast<const void *>(TYPE_SUPPORTS->typesupport_identifier), \
```

Comment:
good point, fixed for connext and connext dynamic in https://github.com/ros2/rmw_connext/pull/219/commits/9c43edf6200347f2bff1983a0e5648d01910b10d